po/zh.po

```diff
@@ -271,7 +271,7 @@ msgstr "%N %V - 文章 '%T'（%u 未读, 共有 %t 篇）"
 msgid ""
 "%N %V - Search results for '%s' (%u unread, %t total)%?F? matching filter "
 "'%F'&?"
-msgstr "%N %V - '%s' 的查询结果 (%u 未读, 共有 %t)%?F? 匹配过滤器'%F'&?"
+msgstr "%N %V - '%s' 的查询结果（%u 未读, 共有 %t）%?F? 匹配过滤器'%F'&?"
```

Comment:
No idea if this is just a bad font selection on my end but it appears that the closing parenthesis is a mix of a parenthesis combined with a space at the end. I even have no clue if such a thing actually exists or not. In case there is a version without the space at the end, that would be the better option here. Otherwise there will be two spaces if a filter expression exists. Feel free to close this if I'm talking rubbish here. :-)

Righto, after looking through all the changes and seeing the same with other punctuation, I reckon that's just the way it is with either my font or how Chinese is typically rendered. Most likely all good, but I leave the comment just in case. You can enlighten us for sure. :-)

Reply:
I think we can remove the space at the start of the description of the filter:

```suggestion
msgstr "%N %V - '%s' 的查询结果（%u 未读, 共有 %t）%?F?匹配过滤器'%F'&?"
```

Then it'd look nice no matter if the filter is defined or not.

Reply:
After having a close look, I found it interesting that Chinese parenthesis  `（）` seems to have different radian with each other. It may be caused by the renderer.

Reply:
Just for reference, on my end it looks like this. I selected the opening parenthesis of your comment above to highlight the parenthesis' left padding.

> ![parentheses](https://github.com/newsboat/newsboat/assets/25375116/752ab48e-bd1c-4c0c-8e69-bcc46379476d)

Quite something to learn here for typography nerds. ;-) I love it!

Reply:
Ah I see. The padding is intended. In monospaced font, all Chinese characters occupy twice the width of English characters, so as the punctuation (except the ellipsis, it's quite long, some times we even use `。。。。。。` in hand-writing). A similar pair to parenthesis is `《》` which we use for titles of books. They also have left and right paddings.